onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -1282,6 +1282,22 @@ TensorrtExecutionProvider::PerThreadContext& TensorrtExecutionProvider::GetPerTh
   return *context;
 }
 
+std::vector<nvinfer1::PreviewFeature> ParseTrtPreviewFeatures(const std::string& str) {
+  std::vector<std::string> featureNames{SplitToStringVec(str, ',')};
+
+  std::vector<nvinfer1::PreviewFeature> previewFeatures;
+  previewFeatures.reserve(featureNames.size());
+  for (auto featureName : featureNames) {
+      if (featureName == "aliasedPluginIO1003") {
```

Comment:
This name is mixed case. Can we use same case like `aliased_plugin_io_10_03` or `ALIASED_PLUGIN_IO_10_03`